app/app.go

```diff
@@ -742,6 +742,20 @@ func (app *GaiaApp) ModuleAccountAddrs() map[string]bool {
 	return modAccAddrs
 }
 
+// BlockedModuleAccountAddrs returns all the app's blocked module account
+// addresses.
+func (app *GaiaApp) BlockedModuleAccountAddrs() map[string]bool {
+	modAccAddrs := app.ModuleAccountAddrs()
+
+	// remove module accounts that are ALLOWED to received funds
+	//
+	// TODO: Blocked on updating to v0.46.x
+	// delete(modAccAddrs, authtypes.NewModuleAddress(grouptypes.ModuleName).String())
```

Comment:
we can uncomment this and merge ? main branch is already in sdk 0.46.

Reply:
Yes but I don't think the changes @alexanderbez made are in the release we're using, it's in `0.46.x` not `rc1`

Reply:
https://github.com/cosmos/gaia/issues/1537

Reply:
ok